app/components/certificate/certificate-display.tsx

```diff
@@ -97,7 +97,7 @@ export default function CertificateDisplay({
         </Tooltip>
       </HStack>
       <Text>{description}</Text>
-      <Accordion allowMultiple>
+      <Accordion allowMultiple paddingY="6">
```

Comment:
Does it make more sense to put this on the root `<Flex>` component, since you're trying to push down all content in here?